### DISCUSSION TITLE
Does this SDK clients use Dataverse web API internally ?

### DISCUSSION THREAD

**Discussion by arjaiswa** [microsoft/PowerPlatform-DataverseServiceClient#371] [Q&A]
1. I am checking whether we can use this SDK for dataverse related operation.
2. Is SDK client using Dataverse API internally?

**Comment by MattB-msft** [comment]
Yes,  however please reach out to me internally.

________________________________
From: Arti Jaiswal ***@***.***>
Sent: Sunday, April 23, 2023 12:20:00 PM
To: microsoft/PowerPlatform-DataverseServiceClient ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [microsoft/PowerPlatform-DataverseServiceClient] Does this SDK clients use Dataverse web API internally ? (Discussion #371)


  1.  I am checking whether we can use this SDK for dataverse related operation.
  2.  Is SDK client using Dataverse API internally?
